python/deltalake/_internal.pyi

```diff
@@ -95,18 +95,34 @@ class Field:
         *,
         nullable: bool = True,
         metadata: Optional[Dict[str, Any]] = None,
-    ) -> None: ...
+    ) -> None:
+        """A named field, with a data type, nullability, and optional metadata."""
     name: str
+    """The field name."""
     type: DataType
+    """The field data type."""
     nullable: bool
+    """The field nullability."""
     metadata: Dict[str, Any]
+    """The field metadata."""
```

Comment:
Does this show up in the API reference correctly? I had never seen this syntax before.

Reply:
Checked again, and it seems to be valid.